icu4c/source/common/loclikely.cpp

```diff
@@ -266,7 +269,8 @@ _uloc_addLikelySubtags(const char* localeID,
     trailing = &localeID[trailingIndex];
     trailingLength = (int32_t)uprv_strlen(trailing);
 
-    CHECK_TRAILING_VARIANT_SIZE(trailing, trailingLength);
+    if (!CHECK_TRAILING_VARIANT_SIZE(trailing, trailingLength)) goto error;
```

Comment:
```suggestion
    if (!CHECK_TRAILING_VARIANT_SIZE(trailing, trailingLength)) { goto error; }
```
... and can we do better now than `goto error`?

Reply:
I'll be coming for `goto` next, that's just easier to do if getting this out of macros first.

Reply:
ok, but braces please?

Reply:
Is spending an hour on running the CI after applying that change really a meaningful use of time for something that's about to be deleted anyway?

Reply:
Applying those changes will trigger a CI run, which takes about an hour. It then leaves the PR with commits that'll need to be squashed before the PR can be merged. Squashing them will trigger a CI run, which takes about an hour again. Is that really a meaningful use of time for something that's about to be replaced anyway?

Reply:
amending the commit would trigger only one round, not two...